### PR TITLE
Updated wine.rb

### PR DESCRIPTION
**Pull request by skiman6010**
Updated wine to 3.13 and enabled X-server support.

### CODE REVIEW

**Comment by uberhacker** [comment]
@skiman6010: Thank you for the update!  There are a ton of things you can enable in wine.  Have you looked through all the options yet?  I think this project could definitely benefit from a few of those.  Any particular reason you disabled freetype support?  If you received an error during the compile process, please post here.  Maybe we can help resolve.

**Comment by JL2210** [comment]
You could use the stable version.

File: packages/wine.rb [hunk 1]
```diff
@@ -4,21 +4,13 @@
...
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/wine-2.10-1-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-    aarch64: 'de6c0bdaf7e2b913a4170588eb1474b34ef48cca4de848aabd8e1b875c6323b4',
-     armv7l: 'de6c0bdaf7e2b913a4170588eb1474b34ef48cca4de848aabd8e1b875c6323b4',
-       i686: 'f741ba56e1d18d26813feab82a2997017ae3325e19cf449c32a61589717178dd',
-     x86_64: '217be92db577ede5038f0bceb1800453bad4e89f2af8f26cdab6afe99384ae2a',
   })
 
   depends_on 'bison'
```
Comment (uberhacker) [inline packages/wine.rb]: Don't we also need `depends_on 'sommelier'`?

Comment (skiman6010) [reply packages/wine.rb]: Oh yeah good call my dude.

**Comment by skiman6010** [comment]
I debated back and forth with myself, but wine moves so quickly and fixes so much in the development version that I think keeping up with the changes would be worth the extra effort to get the most compatibility out of the package.

File: packages/wine.rb [hunk 2]
```diff
@@ -4,33 +4,26 @@
...
   def self.build
     case ARCH
     when "i686", "armv7l", "aarch64"
-      system "./configure --without-x"
+      system "./configure"
     when "x86_64"
-      system "./configure --without-x --enable-win64"
+      system "./configure --enable-win64"
     else
       abort "Error getting your device configuration."
```
Comment (uberhacker) [inline packages/wine.rb]: It might be more appropriate to have a message like:
```
abort "#{ARCH} architecture not supported.".lightred
```

**Comment by skiman6010** [comment]
I mean it compiles fine and it runs with sommelier support. Not sure what we should disable until we run into errors that we find by deploying it to the users. I can't test every use case and exe.

**Comment by uberhacker** [comment]
I'm not taking about disabling anything.  I'm just saying we should include more dependencies, otherwise we could be missing out on some important added functionality.

**Comment by JL2210** [comment]
@skiman6010: Capability, not compatibility. This is another thing that you will have to constantly have to update if you do this, you know. Just a warning.

**Pull request merged**